docs/github/v2/06-remote-builder.md

```diff
@@ -0,0 +1,123 @@
+
+## Basic Remote Builder setup with AWS
+
+### Requirements
+
+- You must have an AWS account setup and ready to create resources.
+- It is suggested you have a workflow setup from the [builder section](builder).
+
+### Create base stack on AWS Cloud Formation
+
+1. Open [this link](https://raw.githubusercontent.com/game-ci/unity-builder/main/dist/cloud-formations/base-setup.yml), right-click and save as a yaml file locally (filename doesn't matter, must end with .yaml).
+2. Open AWS console, navigate to the Cloud Formation service.
+3. Select the option to create a new stack.
+4. Locate the section where you can upload a template. Upload the file you downloaded in step 1. The default values for all other fields will work, usually I name the stack something like `game-ci-base`.
+5. Create the stack and wait for the stack to finish creating. You can delete this stack at any time to cleanup the resources.
+
+_Note:_
+
+- _An AWS Cloud Formation stack is just a group of resources that is created and destroyed together._
+- _The stack we created manages the persistent storage and aws permissions for the builds._
+
+### AWS Credentials
+
+Setup the following as `env` variables for the GitHub build step:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_DEFAULT_REGION` (should be the same AWS region as the base stack e.g `eu-west-2`)
+
+_Note:_
+_This enables us to access AWS resources from the GitHub builder._
+
+### GitHub Access Token
+
+Include the following `with` variable for the GitHub build step:
+
+- `githubToken` (should be a GitHub access token with permission to get repositories)
+
+_Note:_
+_This enables us to get the repository from the AWS build machine._
+
+### Build Step Parameters
+
+Add the following parameters to the build step described in the [builder section](builder):
+
+- `remoteBuildCluster` (should be `aws`)
+- `awsStackName` (should be the name you gave the base stack in aws)
+- `remoteBuildMemory` (the memory the build container should be given)
+- `remoteBuildCpu` (the CPU units the build container should be given)
+
+#### Allowed CPU/Memory combinations
+
+There are some limitations to the CPU and Memory parameters. AWS will only accept the following combinations:
+
+- `0.25 vCPU` - 0.5 GB, 1 GB, 2 GB
+- `0.5 vCPU` - 1 GB, 2 GB, 3 GB, 4 GB
+- `1 vCPU` - 2 GB, 3 GB, 4 GB, 5 GB, 6 GB, 7 GB, 8 GB
+- `2 vCPU` - Between 4 GB and 16 GB in 1-GB increments
+- `4 vCPU` - Between 8 GB and 30 GB in 1-GB increments
+
+Do not include the vCPU or GB suffix. For example:
+
+```yaml
+- remoteBuildMemory: 2
+- remoteBuildCpu: 0.5
+```
+
+This information may go out of date if AWS update this. Please find the latest documentation at the reference link below.
+
+_Reference:_
+[AWS Fargate Documentation, Task Definitions, Task CPU and memory](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html#fargate-task-defs)
```

Comment:
Recommend just linking this under the word `[limitations]` above.

Reply:
There isn't an anchor to the *exact* bit we want to link to. So I wanted to give the full path nice and clearly. Any suggestions?